src/pages/MessagesAndFriends.tsx

```diff
@@ -111,12 +132,14 @@ const MessagesAndFriends = () => {
             />
           </Box>
         )}
-        <Match
-          isMatchModalOpen={isMatchModalOpen}
-          onClose={handleMatchClose}
-          onChat={startChat}
-          friendsAvatar={friendsData.photo[0]?.src}
-        />
+        {isMatchModalOpen && (
```

Comment:
you don't need condition here, you passed prop

Reply:
Removed